packages/amplify_auth_cognito/example/lib/Widgets/ConfirmResetWidget.dart

```diff
@@ -1,33 +1,31 @@
 import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
```

Comment:
There are some weird things happening with indentation in this file. While I was authoring Dart in VS Code recently, I had a fairly bad experience with its automatic formatting. We probably need to get some team wide standards in place. A very basic one: is the indent level 2 or 4 or 8? Right now, I think there are examples across Dart/Kotlin/Swift files of each. I've _assumed_ that 2 is correct for Dart files, and 4 is correct for Kotlin files. But, there are currently many counter examples where Dart has 4 and Kotlin has 2.

Reply:
You are right, dart is 2, kotlin and swift is 4